src/utils/trust/mapper.to.session.ts

```diff
@@ -0,0 +1,19 @@
+import * as Trust from 'model/trust.model';
+import * as Page from 'model/trust.page.model';
+
+const mapDetail = (
+  data: Page.TrustDetails,
```

Comment:
trailing ',' on line 5 (params) and line 18 (exports) can be removed

Reply:
I would prefer to keep
https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

https://blog.logrocket.com/best-practices-using-trailing-commas-javascript/